api/test/json-api.e2e-spec.ts

```diff
@@ -64,4 +107,57 @@ describe('JSON API Specs (e2e)', () => {
       }
     });
   });
+
+  it('should return a object with a "data" prop as a response to a POST request', async () => {
+    const response = await request(app.getHttpServer())
+      .post('/api/v1/projects')
+      .set('Authorization', `Bearer ${jwtToken}`)
+      .send({
+        ...E2E_CONFIG.projects.valid.minimalInGivenAdminArea({
+          countryCode: fakeCountry,
+        }),
+        organizationId: fakeOrganization.id,
+      });
+
+    expect(typeof response.body).toBe('object');
+    expect(Object.keys(response.body)).toHaveLength(1);
+    expect(response.body.hasOwnProperty('data')).toBe(true);
+    expect(typeof response.body.data).toBe('object');
```

Comment:
What do you think about using custom matcher? We should actually test it for each endpoint, right? On top of that, it currently duplicates in a few places.

https://everyday.codes/javascript/create-custom-jest-matchers-to-test-like-a-pro/

Reply:
yep, this could be a nice improvement - except for non-core stuff like pagination and error responses this has not been a problem in practice though, so I'd suggest to create a task for this but only get to it once we're well ahead with stories that unblock frontend work